rotkehlchen/data_import/importers/bittrex.py

```diff
@@ -24,14 +23,27 @@
 if TYPE_CHECKING:
     from collections.abc import Callable
 
-logger = logging.getLogger(__name__)
-log = RotkehlchenLogsAdapter(logger)
+
+class BittrexFileType(Enum):
+    """
+    The type of the file we are parsing
+    """
+    TRADES = 'Trades'
+    TRADES_OLD = 'Trades Old'
+    TRADES_OLDER = 'Trades Older'
```

Comment:
maybe a comment would be required because we will forget 100% what those names are